### PR TITLE
Always update baseline on merge

### PR DESCRIPTION
**Pull request by glennhickey**
This will keep the baseline data level with master.   But it will sweep away traces of regressions that get merged in.  @adamnovak what do you think about this?

### CODE REVIEW

**Comment by adamnovak** [comment]
I already pushed in a fix for the problem we had where the score worsening
check could fail if there was no baseline yet. So we hopefully wouldn't
need this for that. But it probably does make sense for the baseline to
always be the last master run, because we don't want master to fail forever
if we do end up lowering something below baseline.

File: jenkins/jenkins.sh [hunk 1]
```diff
@@ -236,10 +236,10 @@ then
     tar czf "${VG_VERSION}_output.tar.gz" vgci-work test-report.xml jenkins/vgci.py jenkins/jenkins.sh vgci_cfg.tsv
     aws s3 cp --acl public-read "${VG_VERSION}_output.tar.gz" s3://cgl-pipeline-inputs/vg_cgl/vg_ci/jenkins_output_archives/
 
-    # if success and we're merging the PR (and not just testing it), we publish results to the baseline
-    if [ "$PYRET" -eq 0 ] && [ -z ${ghprbActualCommit} ]
+    # if we're merging the PR (and not just testing it), we publish results to the baseline
+    [ -z ${ghprbActualCommit} ]
```
Comment (adamnovak) [inline jenkins/jenkins.sh]: @glennhickey Looks like you dropped the if; I think baseline might be getting updated from PRs now? Or maybe never updated?

**Comment by glennhickey** [comment]
I blame jet lag.  It's been fixed though:
https://github.com/vgteam/vg/pull/1075

File: jenkins/jenkins.sh (hunk 1 above)
Comment (adamnovak) [reply jenkins/jenkins.sh]: Also looks like you fixed it yesterday.

**Pull request merged**